examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<blockquote>to possibly experiment</blockquote><p>"to possibly experience" I guess?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>